Snowdog_Components/components/Organisms/messages/messages.hbs

```diff
@@ -1,5 +1,9 @@
 <div
-    class="messages {{ class }}"
+    class="
+        messages
```

Comment:
now it works as expected. I have 2 requests:
- please move `messages` class to a separate div to match `Magento_Theme/templates/messages.phtml`
- add `messages--hidden-icons` variant to fractal